src/ProjectTemplates/Web.ProjectTemplates/content/BlazorWeb-CSharp/BlazorWeb-CSharp/Components/Pages/NotFound.razor

```diff
@@ -0,0 +1,4 @@
+﻿@page "/not-found"
+
+<h3>Not Found</h3>
+<p>Sorry, the content you are looking for does not exist.</p>
```

Comment:
Is this the same text as the one in the NotFound fragment?

Reply:
No. There is no `NotFound` in the template, see:
https://github.com/dotnet/aspnetcore/blob/8ddf53d847a8ce1cb0e0787f465ad1aa3ce3ecf1/src/ProjectTemplates/Web.ProjectTemplates/content/BlazorWeb-CSharp/BlazorWeb-CSharp/Components/Routes.razor#L9

Reply:
Fair enough, I was just trying to make sure that we were happy with the message on the template